cmd/release.go

```diff
@@ -240,11 +241,17 @@ func getChangelog(version string, rc io.ReadCloser) (string, string) {
 		s       []string
 		header  string
 		reading bool
+
+		releaseHeaderPattern = "## " + version + " / \\d\\d\\d\\d-\\d\\d-\\d\\d"
```

Comment:
Might be easier to read:

```
releaseHeaderPattern = `## " + version + " / \d{4}-\d{2}-\d{2}`
```

Technically you'd also have to escape the dots in the version, but 🤷

Reply:
fixed, thanks!